### PR TITLE
Added adult keywords

### PR DESCRIPTION
**Pull request by robynk**

### CODE REVIEW

**Comment by kane-c** [comment]
Same here - need commas at the end of those lines.

File: src/pages/personalisation/Age.js [hunk 1]
```diff
@@ -24,16 +24,19 @@ export default class Age extends BaseQuestion {
...
                 ],
             }),
             "40 to 54": append({
+                q: "(middle-aged adults)"
                 age_groups: [
                     "middleageadult",
                 ],
             }),
             "55 to 64": append({
+                q: "(pre-retirement aged"
```
Comment (kane-c) [inline src/pages/personalisation/Age.js]: Missing closing `)`

**Comment by kane-c** [comment]
Also as far as I can tell, the query param `age_groups` should actually be `age_group`

**Comment by DanielHeath** [comment]
Merged into https://github.com/ask-izzy/ask-izzy/pull/365

**Pull request closed without merging**